tox.ini

```diff
@@ -1,35 +1,80 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
+minversion = 3.18
 envlist =
-    py27,py35,py36,py37,py38,pypy,pypy3,coverage,docs
-# Note: if you add new Python versions, please add them to
-# [testenv:coverage] depends as well!
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    docs
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
-    coverage run -p -m zope.testrunner --test-path=src []
+    zope-testrunner --test-path=src {posargs:-vc}
 extras =
     test
-deps =
-    coverage
-setenv =
-    ZOPE_INTERFACE_STRICT_IRO=1
```

Comment:
Is this variable no longer needed?

It can be added via `.meta.toml` like in https://github.com/zopefoundation/zope.formlib/blob/7c300084e79d33bec9143cc3a4bff4f342a9c7a3/.meta.toml#L28-L31

Reply:
Excellent, thank you.

Reply:
Hmm, that doesn't seem to do anything. And I noticed that [zope.formlib's tox.ini](https://github.com/zopefoundation/zope.formlib/blob/master/tox.ini) doesn't contain that setting either. I'm guessing it's in the wrong section?

Reply:
Yeah, it needs to be in `[tox]`.

Reply:
Good catch. Fixing it in zope.formlib in https://github.com/zopefoundation/zope.formlib/pull/38